config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
-  root 'cards#index'
+  resources :cards
+  root 'pages#welcome'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.